examples/keep-alive-framework/keep-alive-framework-app.json

```diff
@@ -0,0 +1,60 @@
+{
+  "env": {
+  },
+  "id": "/usi/keep-alive-framework",
+  "backoffFactor": 1.15,
+  "backoffSeconds": 1,
+  "cmd": "export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/jre/); export JAVA_HOME=${JAVA_HOME%/}; export PATH=$(ls -d $JAVA_HOME/bin):$PATH && export JAVA_OPTS=\"-Xms256M -Xmx512M\"; ./keep-alive-framework-0.1/bin/keep-alive-framework https://master.mesos https://leader.mesos:5050 $MESOS_SANDBOX/.ssl/ca-bundle.crt $MESOS_SANDBOX/private_key.pem strict-usi",
+  "container": {
+    "type": "MESOS",
+    "volumes": [
+      {
+        "containerPath": "private_key.pem",
+        "secret": "private_key"
+      }
+    ]
+  },
+  "cpus": 0.5,
+  "disk": 0,
+  "fetch": [
+    {
+      "uri": "https://s3-us-west-2.amazonaws.com/usi-builds/mesosphere/usi/artifacts/keep-alive-framework-0.1.tar",
+      "extract": true,
+      "executable": false,
+      "cache": false
+    },
+    {
+      "uri": "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz",
```

Comment:
we're moving away from the Oracle jdk; maybe we should just have this run on openjdk?

Reply:
Good point. This should be Amazon Corretto to be save.